airbyte-commons-worker/src/main/java/io/airbyte/workers/WorkerUtils.java

```diff
@@ -140,6 +142,12 @@ private static Optional<AirbyteTraceMessage> getTraceMessageFromMessagesByType(f
         .findFirst();
   }
 
+  public static Boolean getDidControlMessageChangeConfig(final JsonNode initialConfigJson, final AirbyteControlConnectorConfigMessage configMessage) {
+    final Config newConfig = configMessage.getConfig();
+    final JsonNode newConfigJson = Jsons.jsonNode(newConfig);
+    return !initialConfigJson.equals(newConfigJson);
```

Comment:
~~This is OK, but calling out that this would would fail if the config JSON was re-ordered by the connector.  However, the connector shouldn't just re-order it's config without a good reason...~~

Actually, `jackson.equals()` should be positionally independant!